pkg/controller/pxcbackup/utils_test.go

```diff
@@ -0,0 +1,60 @@
+package pxcbackup
+
+import (
+	"math/rand"
+	"reflect"
+	"strconv"
+	"strings"
+	"testing"
+	"testing/quick"
+)
+
+type finalizer string
+
+func (finalizer) Generate(r *rand.Rand, size int) reflect.Value {
+	return reflect.ValueOf(finalizer(strconv.FormatInt(r.Int63(), 2)))
+}
+
+func TestRemoveStringsFromSlice(t *testing.T) {
```

Comment:
I don't understand what are we testing here?

Reply:
the function
https://github.com/percona/percona-xtradb-cluster-operator/blob/693189686ad2a5a06e3d2f26f1ae81b0983bef4b/pkg/controller/pxcbackup/utils.go#L4
and that a caller can rely on the fn behaviour like here
https://github.com/percona/percona-xtradb-cluster-operator/blob/693189686ad2a5a06e3d2f26f1ae81b0983bef4b/pkg/controller/pxcbackup/controller.go#L260